package.json

```diff
@@ -1,43 +1,54 @@
 {
   "name": "@chainsafesystems/ssz",
-  "version": "1.6.0",
-  "description": "A pure JS implementation of simple serialize",
-  "main": "src/index.js",
+  "version": "0.1.0",
```

Comment:
can we bump the version?

Reply:
Posting what we talked about earlier: We're planning on publishing under a new name `@chainsafe/ssz` and this is a clean rewrite, so we're resetting the version.